apps/web/components/v2/eventtype/EventLimitsTab.tsx

```diff
@@ -1,23 +1,34 @@
 import { useAutoAnimate } from "@formkit/auto-animate/react";
 import * as RadioGroup from "@radix-ui/react-radio-group";
 import { EventTypeSetupInfered, FormValues } from "pages/event-types/[type]";
-import { useState } from "react";
+import { useEffect, useRef, useState } from "react";
 import { useFormContext, Controller, useWatch } from "react-hook-form";
 
 import { classNames } from "@calcom/lib";
+import convertToNewDurationType from "@calcom/lib/convertToNewDurationType";
+import findDurationType from "@calcom/lib/findDurationType";
 import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { PeriodType } from "@calcom/prisma/client";
 import type { BookingLimit } from "@calcom/types/Calendar";
 import { Icon } from "@calcom/ui";
 import { Button } from "@calcom/ui/components";
-import { Label, Input, MinutesField } from "@calcom/ui/components/form";
+import { Label, Input, InputField } from "@calcom/ui/components/form";
 import { Select, SettingsToggle } from "@calcom/ui/v2";
 import DateRangePicker from "@calcom/ui/v2/core/form/date-range-picker/DateRangePicker";
 
+export let CurrentDurationType: string | "minutes" | "hours" | "days";
```

Comment:
@om-ray Why are we exactly changing this variable and importing this elsewhere? Changing this variable from within a component and reusing it somewhere else is not a react standard, this variable might not update. Could we also use a ref for this? 

If we need this value in the parent component we should probably work with a change handler that we pass in as a prop I think?